x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_lifecycle/summary.tsx

```diff
@@ -28,6 +28,8 @@ export function RetentionSummary({ definition }: { definition: IngestStreamGetRe
             defaultMessage: 'Retention summary',
           })}
         </h5>
+      </EuiText>
+      <EuiText size="s" color="subdued">
```

Comment:
Add `component="p"` to EuiText and remove the <p> around summary to have a single DOM node rendered